pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -818,7 +822,7 @@ func (b *Build) rebasePod(buildContext BuildContext, images BuildPodImages) (*co
 					Resources: b.Spec.Resources,
 					Args: args(a(
 						"--run-image",
-						buildContext.BuildPodBuilderConfig.RunImage,
+						runImage,
```

Comment:
I think we also want to pass the run image until the non-rebase build.

Reply:
Yeah, I didn't do it just to get early feedback on it, but I will add it

Reply:
Working on this, but got the following error on my first attempt:

![Screen Shot 2022-07-12 at 12 28 04 PM](https://user-images.githubusercontent.com/1181799/178558678-1b3b982d-093e-48ab-8228-1d0f97c8925f.png)

Reply:
this is suggesting that the value used for the `-run-image` flag argument is unset

Reply:
it appears `-run-image` was moved from exporter to analyzer in platform api version 0.7: https://buildpacks.io/docs/reference/spec/migration/platform-api-0.6-0.7/

Maybe we don't pass the run image to the non-rebase builds for now until we have implemented this